.github/workflows/changelog.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Changelog
+
+on:
+  workflow_dispatch:
+  schedule:
+    # every Monday morning
+    - cron:  '0 2 * * 1'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  update-dependencies:
+    strategy:
+      fail-fast: false
+    runs-on: 'ubuntu-latest'
+    name: Update Changelog
+
+    steps:
+      - name: Checkout ${{ github.sha	}}
+        uses: actions/checkout@v2
+
+      - name: Update changelog
+        run: |
+          UPDATES=$(git log --after=$(date -d '7 days ago' +%Y-%m-%dT%00:00:00) --pretty=format:%s)
+          PRE_PROCESSED=$(printf '%s\n' "$UPDATES" | sed 's/\\/&&/g;s/^[[:blank:]]/\\&/;s/$/\\/')
+          sed -i "/^## \[Unreleased\]/a\\$PRE_PROCESSED" CHANGELOG.md
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.OSC_ROBOT_GH_PUB_REPO_TOKEN }}
+          commit-message: update dependencies
```

Comment:
@johrstrom I think maybe copy paste mistake here?

Reply:
Yep! I caught the same in the PR it opened. I have a fix in #1405.